### PR TITLE
docs: update client v3's README

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Review by spzala — COMMENTED** [review]
I am not sure what you are trying to do here?? Closing the PR, please reopen with more explanation if needed. Thanks!

**Comment by juzhiyuan** [comment]
The original link is broken indeed

**Comment by spzala** [comment]
> The original link is broken indeed

Hmm..I guess I might have it cached. Reopening it. Thanks for the quick reply!

File: client/v3/README.md [hunk 1]
```diff
@@ -89,4 +89,4 @@ Client request size limit is configurable via `clientv3.Config.MaxCallSendMsgSiz
 
 ## Examples
 
-More code examples can be found at [GoDoc](https://godoc.org/go.etcd.io/etcd/clientv3).
+More code examples can be found at [GoDoc](https://godoc.org/github.com/rpcxio/etcd/clientv3).
```
Comment (ptabor) [inline client/v3/README.md:92]: The working path is: 

https://pkg.go.dev/go.etcd.io/etcd/client/v3

If you really want to point at examples, I would recommend a direct link to: 

https://github.com/etcd-io/etcd/tree/master/tests/integration/clientv3/examples

**Review by ptabor — CHANGES_REQUESTED** [review]

**Review by ptabor — APPROVED** [review]

**Pull request merged**